### PR TITLE
[CCXDEV-11599] Add DVO extractor tests

### PR DESCRIPTION
**Pull request by JiriPapousek**
# Description

Added tests for DVO extractor and refactor test steps related to both ccx-messaging based services.

## Type of change

- New test feature file
- Non-breaking change in test steps implementation
- Refactor (refactoring code, removing useless files)
- Configuration update

## Testing steps

Run both DVO extractor and SHA extractor tests using `run_in_docker.sh` script and locally.

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [x] new tests can be executed both locally and within docker container
* [x] new tests have been included in scenario list (make update-scenarios)

### CODE REVIEW

File: features/steps/ccx_messaging.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    """Check if SHA extractor service has been started."""
+    service_name = transform_service_name(service)
+    assert (
+        not hasattr(context, "services") or service_name not in context.services.keys()
+    )
+
+
+@given(
+    'Kafka broker is started on host and port specified '
+    'in {service} configuration "{compression_var}"',
```
Comment (epapbak) [inline features/steps/ccx_messaging.py:45]: I'm lost about what this compression_var is and why the configuration file would be compressed

Comment (juandspy) [reply features/steps/ccx_messaging.py:45]: I think it's used to start the service with/without compression like in
```
    Given Kafka broker is started on host and port specified in SHA extractor configuration "no-compression"
```

Comment (JiriPapousek) [reply features/steps/ccx_messaging.py:45]: This one was part of SHA extractor tests done by Jakub. But yes, Juan is right, there are two configurations, one with enabled message compressions and one without.

Comment (epapbak) [reply features/steps/ccx_messaging.py:45]: It's pretty error prone (an entire config file is loaded just to enable compression), but we can solve that in a different PR,

**Review by epapbak — APPROVED** [review]

**Review by juandspy — APPROVED** [review]

**Pull request merged**